content/1.2/docs/serverless-workflows/troubleshooting.md

```diff
@@ -94,3 +94,27 @@ a CA which is not available to the workflow. The error in the workflow pod log u
 
 **Solution:**
 When deploying a workflow in a namespace other than the one where Sonataflow services are running (e.g., sonataflow-infra), there are essential steps to follow to enable persistence and connectivity for the workflow. See the following [steps](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/release-1.2/README.md#additional-workflow-namespaces).
+
+### Problem: sonataflow-platform-data-index-service pods can't connect to the database on startup
```

Comment:
Just asking for the sake of asking: the DI pods was not able to connect to the PSQL DB running on the sonataflow-infra NS, but was the psql pod running? If the DI and JS pods are started at the same time as the PSQL, they should wait and restart until the PSQL DB pod is available

Reply:
The PSQL post was started and was running before deploying the operator, so it wasn't the case.
Do you suggest adding also this possible solution to the list?

Reply:
Not a solution but a warning; like the PSQL DB pod may be starting so wait a little before DI and JS pods (and then workflows) pods are ready